tox.ini

```diff
@@ -53,8 +53,8 @@ setenv =
   COV_CORE_CONFIG = {toxinidir}/.coveragerc
   COV_CORE_DATAFILE = {toxinidir}/.coverage
   COV_CORE_SOURCE =
-  PYTHONPATH = packaging/
   SEGFAULT_SIGNALS = all
+  !just-pytest: PYTHONPATH = packaging/
```

Comment:
Could you explain the motivation here? What does this affect? I want to redesign the tox setup at some point, and if I accept this change, I need to understand whether it needs to be kept later on.

Reply:
It's about %tox in rpm spec file. %tox sets PYTHONPATH and execute the test in current environmen instead of creating new vitual environment so the packages that are installed in buildroot are tested. However, just-pytest inherits setenv from default environment thus always overriding PYTHONPATH and therefore the test fails to import pylibsshext.

Reply:
...because PYTHONPATH is packaging/ instead of site packages from buildroot.

Reply:
Would this work?
```suggestion
  PYTHONPATH = {env:PYTHONPATH:packaging/}
```

Reply:
Or maybe
```suggestion
  PYTHONPATH = packaging/:{env:PYTHONPATH:}
```

Reply:
Removing `packaging/` will make the first command fail. It's prepended with `-` so `tox` will ignore its outcome. Nevertheless, it won't generate C-files with tracing information if it stops being importable.